tests/config_id/enc/sign_kss.conf

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+# Enclave settings:
+Debug=1
+NumHeapPages=1024
+NumStackPages=1024
+NumTCS=2
+ProductID=1
+SecurityVersion=1
+ExtendedProductID=25419627-14f6-4625-9329-cf5f10a57feb
```

Comment:
Are this the following configurations used anywhere?

Reply:
they are not used anywhere for config_id/config svn test purpose